Fabric.Identity.IntegrationTests/ServiceTests/IdentityConfigurationProviderTests.cs

```diff
@@ -23,10 +25,14 @@ public void GetAppConfiguration_ReturnsAppConfig_Success()
             File.WriteAllText(Path.Combine(directory.FullName, "appsettings.json"), appSettingsJson);
             var mockCertificateService = GetMockCertificateService(privateKey);
             var decryptionService = new DecryptionService(mockCertificateService);
-            var identityConfigurationProvider = new IdentityConfigurationProvider();
+            var config = new ConfigurationBuilder()
+                .AddJsonFile(Path.Combine(directory.FullName, "appsettings.json"))
+                .Build();
+            var identityConfigurationProvider = new IdentityConfigurationProvider(config);
+            
 
             // Act
-            var appConfig = identityConfigurationProvider.GetAppConfiguration(directory.FullName, decryptionService);
```

Comment:
I noticed in the code that this string param was removed, is this not necessary in our setup to distinguish IdentityProviders?